db/schema.rb

```diff
@@ -1440,7 +1440,6 @@
   end
 
   add_foreign_key "api_docs_services", "services"
-  add_foreign_key "event_store_events", "accounts", column: "provider_id", on_delete: :cascade
```

Comment:
If this is correct, this also implies telling ops to run the migration on SaaS 😄